rotkehlchen/tests/data_migrations/test_migrations.py

```diff
@@ -675,6 +706,9 @@ def test_migration_18(rotkehlchen_api_server: 'APIServer') -> None:
             tx_hash=tx_hash,
         )
 
+    # mark a few assets as spam
+    tokens = _populate_spam_tokens(rotki.data)
```

Comment:
Since this function is only used here why not use it in here anyway? Makes it easier to followthe logic